03_launch_mgmt_cluster.sh

```diff
@@ -55,10 +55,12 @@ CAPM3_RUN_LOCAL="${CAPM3_RUN_LOCAL:-false}"
 
 if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
   IRONIC_HOST="${PROVISIONING_URL_HOST}"
-  BMO_CONFIG="ironic-outside-config"
```

Comment:
This cannot be changed yet. you still need to deploy the old kustomization to keep things working

Reply:
Done